source/fundamentals/indexes.txt

```diff
@@ -0,0 +1,268 @@
+collection scans are slow and can negatively affect the performance of your application. By using an index to limit the
+number of documents MongoDB scans, queries can be more efficient and return faster.
+
+
+Query Coverage and Performance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you execute a query against MongoDB, your query can include three parts:
+
+- query criteria that specify field(s) and value(s) you are looking for
+- options that affect the query's execution (e.g. read concern)
+- projection criteria to specify the fields MongoDB should return (optional)
+
+When all the fields specified in the query criteria and projection of a query are indexed, MongoDB returns results
+directly from the index without scanning any documents in the collection or loading them into memory.
+
+For additional information on how to ensure your index covers your query criteria and projection, see the MongoDB manual
+articles on :manual:`query coverage </core/query-optimization/#read-operations-covered-query>`.
+
+Operational Considerations
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To improve query performance, build indexes on fields that appear often in your application's queries and operations
+that return sorted results. Each index that you add consumes disk space and memory when active so you should track index
+memory and disk usage for capacity planning. In addition, when a write operation updates an indexed field, MongoDB also
+has to update the related index.
+
+For more information on designing your data model and choosing indexes appropriate for your application, see the MongoDB
+server documentation on :manual:`Indexing Strategies </applications/indexes>` and
+:manual:`Data Modeling and Indexes </core/data-model-operations/#data-model-indexes>`.
+
+Index Types
+-----------
+
+MongoDB supports a number of different index types to support querying your data. The following sections describe the
+most common index types and provide sample code for creating each index type.
+
+.. tip::
+
+   The MongoDB Java Driver provides the :java-core-api:`Indexes </com/mongodb/client/model/Indexes.html>` class that
+   includes static factory methods to create index specification documents for different MongoDB Index key types.
+
+The following examples use the
+:java-sync-api:`createIndex </com/mongodb/client/MongoCollection.html#createIndex(org.bson.conversions.Bson,com.mongodb.client.model.IndexOptions)>`
+to create various indexes, and the following setup:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/IndexPage.java
+   :language: java
+   :dedent:
+   :start-after: begin declaration
+   :end-before: end declaration
+
+
+Single Field and Compound Indexes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Single Field Indexes
+++++++++++++++++++++
+
+**Single field indexes** are indexes that improve performance for queries that specify ascending or descending sort
```

Comment:
This is the first mention of sort criteria.  And it's also not the only or even primary use of a single field index.  For example, the `_id` index.

Reply:
We do include mention of the `_id_` index further down in the page, but I added another callout here. I also added a short tip to let readers know indexes help other operations.

Reply:
+1 for the sentiment that sort order is not the only use case for a single field index, filtering being the primary example.